orbitx/flight_gui.py

```diff
@@ -428,7 +431,14 @@ def build_menu(*, choices=None, bind=None,
     def draw(self, physical_state_to_draw):
         self._last_physical_state = physical_state_to_draw
         # Have to reset origin, reference, and target with new positions
-        self._set_origin(self._origin.name)
+        # TODO: Trails are reset when change the origin, can we delete it?
```

Comment:
Patrick, can I delete 441? This code was originally added before you separate ref and origin, which fixed various issues we had. After reviewing the code, this became unnecessary, (and when _set_origin, trails are deleted)

Reply:
As long as the position of the origin is updated every tick, sounds good. I can't check since I'm on my phone, but if you want to stop the trail being cleared but keep the origin updating every tick, you may have to separate these behaviours.

Reply:
I just checked, removing this line will not update the origin. It's not immediately obvious, but after a long runtime, we get graphical glitches again: 
![image](https://user-images.githubusercontent.com/1498589/52363071-ed8fa180-2a0f-11e9-99a1-c91e3ff5ce45.png)

We'll have to decouple updating the origin and clearing trails. I'll try to make a quick commit.